package/src/index.ts

```diff
@@ -70,6 +92,260 @@ export class CandleClient {
   }): Promise<string> {
     return this.candle.executeTool(tool);
   }
+
+  public async getAssetAccounts(
+    query: AssetAccountQuery = {
+      linkedAccountIDs: undefined,
+      assetKind: undefined,
+    }
+  ): Promise<AssetAccount[]> {
+    const accounts = await this.candle.getAssetAccounts(query);
+    return accounts.map(this.convertToAssetAccount);
+  }
+
+  public async getTrades(
+    query: {
+      gainedAssetKind?: TradeQueryAssetKind;
+      lostAssetKind?: TradeQueryAssetKind;
+      counterpartyKind?: "merchant" | "user" | "service";
+    } & TradeQuery = {
+      counterpartyKind: undefined,
+      gainedAssetKind: undefined,
+      lostAssetKind: undefined,
+      linkedAccountIDs: undefined,
+      dateTimeSpan: undefined,
+    }
+  ): Promise<
+    {
+      dateTime: string;
+      state: TradeState;
+      counterparty: Counterparty;
+      lost: TradeAsset;
+      gained: TradeAsset;
+    }[]
+  > {
+    const trades = await this.candle.getTrades(query);
+    return trades.map((t) => ({
+      dateTime: t.dateTime,
+      state: t.state,
+      counterparty: this.convertToCounterparty(t.counterparty),
+      lost: this.convertTradeAsset(t.lost)!,
+      gained: this.convertTradeAsset(t.gained)!,
```

Comment:
#req we should never use `!`, you're just bypassing the compiler (it doesn't validate that the expression is non-nil like Swift)